attractions/utils/datetime-utils.js

```diff
@@ -215,12 +215,12 @@ export function datesLessEqual(date1, date2) {
  */
 export function getCalendar(month, year, firstWeekday) {
   const calendar = [];
-  const dayCursor = new Date();
-  dayCursor.setFullYear(year, month, 1);
+  const dayCursor = new Date(1970, 0, 1);
+  dayCursor.setFullYear(year, month);
```

Comment:
Couldn't these 2 lines be combined as such:
```suggestion
  const dayCursor = new Date(year, month, 1);
```
?

Reply:
Thanks, I did not know about that equivalency! Committed your suggested change.